README.md

```diff
@@ -1,7 +1,50 @@
 # sknnr-spatial
 
-[![Build status](https://github.com/lemma-osu/sknnr-spatial/actions/workflows/ci.yaml/badge.svg)](https://github.com/lemma-osu/sknnr-spatial/actions/workflows/ci.yaml)
+[![Build status](https://github.com/lemma-osu/sknnr-spatial/actions/workflows/ci.yaml/badge.svg)](https://github.com/lemma-osu/sknnr-spatial/actions/workflows/ci.yaml) [![Documentation status](https://readthedocs.org/projects/sknnr-spatial/badge/?version=latest)](https://sknnr-spatial.readthedocs.io/)
 
-Spatial predictions using [scikit-learn kNN regression](https://github.com/lemma-osu/sknnr).
+> ⚠️ **WARNING: sknnr-spatial is in active development!** ⚠️
 
-> ⚠️ **WARNING: sknnr-spatial is in active development!** ⚠️
\ No newline at end of file
+## Features
+
+- 🗺️ Raster predictions from [scikit-learn](https://scikit-learn.org/stable/) estimators 
+- ⚡ Parallelized functions + larger-than-memory data using [Dask](https://www.dask.org/)
+- 🌐 Automatic handling of projections, band names, and masks
```

Comment:
I think I'm going to be embarrassed for not remembering, but we don't have explicit projection support, correct?  That is, a user couldn't create an `xarray.DataArray` from two rasters with different projections into a `xarray.Dataset` and it would handle the projection on-the-fly, right?  Or maybe I'm wrong here?

I think this means that projection information from the input rasters is carried through to the output raster as well, correct?  Perhaps "automatic handling of spatial references, ..."?